src/Umbraco.Cms.Api.Management/Controllers/RecycleBin/RecycleBinControllerBase.cs

```diff
@@ -23,6 +24,9 @@ protected RecycleBinControllerBase(IEntityService entityService)
 
     protected abstract Guid RecycleBinRootKey { get; }
 
+    protected virtual Ordering ItemOrdering => Ordering.By(nameof(Infrastructure.Persistence.Dtos.NodeDto.Text));
```

Comment:
Do we need this as a `protected` field?  I couldn't see that we are overriding it anywhere.  If we aren't, then could it just be a hardcoded value in `GetSiblingEntities`.